### ISSUE TITLE
Execution fails when multiple queries exist

### ISSUE THREAD

**Issue by Hadakajime** [apollographql/apollo-client-devtools#444]
Running Firefox 86.0 with Apollo client devtools 2.3.5 on PopOs 20.10

1. If multiple queries exist in the query box, then execution fails. If you comment out every query except the one you want, then it runs.

The failing result is just:
`[object Object]`

Works running `query user`:

```
# query j {
#   job(id:1) {
#     id
#   }
# }

# query hiredCandidates {
#   hiredCandidates(companyId: 3, confirmed: true) {
#     id
#   }
# }

query user {
  user(id: 81){
    id
  }
}

# query usernameIsAvailable {
#   usernameIsAvailable(username: "aperture87@demo.com") {
#     success
#     error
#   }
# }
```
```
{
  "data": {
    "user": {
      "id": 81,
      "__typename": "User"
    }
  },
  "loading": false,
  "networkStatus": 7,
  "stale": false
}
```

Fails running `query user`:

```
# que
ry j {
#   job(id:1) {
#     id
#   }
# }

# query hiredCandidates {
#   hiredCandidates(companyId: 3, confirmed: true) {
#     id
#   }
# }

query user {
  user(id: 81){
    id
  }
}

query usernameIsAvailable {
  usernameIsAvailable(username: "aperture87@demo.com") {
    success
    error
  }
}
```
```
[object Object]
```

2. Also, if you accidentally hit the Prettify button, it will delete all your commented queries.
Win win.

3. Lastly, when running with only a single query, the Play button still thinks the query is executing, even after the result has been returned. It shows  □ until you click it again.

**Comment by hwillson** [comment]
@Hadakajime all of these items should be fixed in our latest version 3.0, but we're unfortunately still held up in review so it hasn't been made officially available yet. You can try the latest version out using these [local build/install instructions](https://github.com/apollographql/apollo-client-devtools/blob/main/README.md#development-with-web-ext--webextwebpackplugin), if you're interested.

**Comment by Hadakajime** [comment]
Awesome. Thanks.
